### DISCUSSION TITLE
Is it possible to use conditional fields on settings page?

### DISCUSSION THREAD

**Discussion by michaeljcoyne** [area17/twill#1267] [General]
Using the example from the docs - placing the below code the my settings file - content field section:

` @formConnectedFields([
    'fieldName' => 'vertical_article',
    'fieldValues' => true,
    'renderForBlocks' => true # (depending on regular form vs block form)
    ])
    @formField('medias', [
    'name' => 'vertical_image',
    'label' => 'Vertical Image',
    ])
    @endformConnectedFields`

This produces an error
ReferenceError: fieldName is not defined

Is it possible to use conitonal fields in the settings page?

**Comment by pboivin** [comment]
Hi @michaeljcoyne, I think you would need `'renderForBlocks' => false` in the context of a settings page.

**Comment by michaeljcoyne** [comment]
Ok thought I tried that but today it works.
Thanks
